src/quest_manager/forms.py

```diff
@@ -81,9 +81,9 @@ class Meta:
         }
 
         widgets = {
-            'instructions': SummernoteInplaceWidget(),
-            'submission_details': SummernoteInplaceWidget(),
-            'instructor_notes': SummernoteInplaceWidget(),
+            'instructions': ByteDeckSummernoteInplaceWidget(),
+            'submission_details': ByteDeckSummernoteInplaceWidget(),
+            'instructor_notes': ByteDeckSummernoteInplaceWidget(),
```

Comment:
These three fields will use the the "advanced" (unsafe) widget, once you get there.